tests/NimblePros.MediatR.Contrib.Test/Behaviors/CachingBehaviorTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using MediatR;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
+
+namespace NimblePros.MediatR.Contrib.Test.Behaviors;
+
+public class CachingBehaviorTests
+{
+  [Fact]
+  public async Task Should_ReturnSuccess_GivenValidInputWithCacheMiss()
+  {
+    // Arrange
+    var nextMock = new Mock<INext>();
+    var memoryCacheMock = new Mock<IMemoryCache>();
+    var loggerMock = new Mock<ILogger<Mediator>>();
+    var cacheEntryMock = new Mock<ICacheEntry>();
```

Comment:
I'd pull these up into private fields since they're used in every test in this class.